src/Umbraco.Web.UI.Client/src/packages/tiptap/components/input-tiptap/input-tiptap.element.ts

```diff
@@ -236,6 +245,7 @@ export class UmbInputTiptapElement extends UmbFormControlMixin<string, typeof Um
 		return html`
 			<umb-tiptap-statusbar
 				data-mark="tiptap-statusbar"
+				aria-label=${this.#getLabel('status bar')}
```

Comment:
The same as toolbar, the statusbar element isn't an input itself and doesn't require an `aria-label` attribute.